tree/dataframe/test/dataframe_cloning.cxx

```diff
@@ -377,31 +409,27 @@ TEST(RDataFrameCloning, ChangeSpec)
    // ChangeSpec. Do this by `Take`ing the values of column x for
    // every partition and checking that they correspond to the values
    // in the ranges defined by globalRanges.
-   std::vector<std::vector<ULong64_t>> expectedOutputs;
-   expectedOutputs.reserve(6);
-   for (unsigned i = 0; i < 6; i++) {
+   std::vector<std::vector<std::uint64_t>> expectedOutputs;
```

Comment:
If possible, I would not change this int type and leave ROOT's platform independent typedef.